src/api/providers/openai.ts

```diff
@@ -22,6 +23,9 @@ export class OpenAiHandler implements ApiHandler {
 			this.client = new OpenAI({
 				baseURL: this.options.openAiBaseUrl,
 				apiKey: this.options.openAiApiKey,
+				defaultHeaders: {
```

Comment:
Did you consider adding to `AzureOpenAI` a few lines up?

Reply:
thank you for the suggestion, done!